lib/sequelize.js

```diff
@@ -667,22 +632,16 @@ module.exports = (function() {
    *
    * @see {Model#build} for more information about callee.
    */
-  Sequelize.prototype.query = function(sql, callee, options, replacements) {
+  Sequelize.prototype.query = function(sql, callee, options) {
```

Comment:
I'll help you if you want me too. I think it's really good to get rid of all the old stuff (besides the original issue that required some of these changes).